rubocop.yml

```diff
@@ -326,6 +326,7 @@ Naming/InclusiveLanguage:
         - 'blob/master/'
         - 'origin/master'
         - 'mastercard'
+        - 'webmaster'
```

Comment:
I opened up the definition of the class and validated that strings in the `AllowedRegex` list here are matched ignoring case. Which is what we want.